File: JavaPatternPrograms/Symbol Patterns/Pattern 76.java
class PyramidSymbol
{
    public static void main(String args[])
    {
        int p_height = 5;

        for (int i = 1; i <= p_height; i++)
        {
            for (int k = p_height - 1; k >= i; k--)
            {
                System.out.print(" ");
            }
            for (int j = 1; j <= i; j++)
            {
                if (i % 2 == 0)
                {
                    System.out.print("* "); // space after *
                }
                else
                {
                    System.out.print("+ "); // space after +
                }
            }
            System.out.println("");
        }
    }
}

File: PythonPatternPrograms/SymbolPatterns/Pattern 76.py
p_height=5
for x in range(1, p_height+1):
    for y in range(p_height-1, x-1, -1):
        print(" ", end="")
    for z in range(1, x + 1):
        if x%2==0:
            print("* ", end="")
        else:
            print("+ ", end="")
    print()